documentation/AuthorizationService.md

```diff
@@ -1,123 +1,245 @@
+For onboarded solution clients, the client information contains: 
++ The client name
++ The solution id
+
+For web preferences consumer clients, the client information contains: 
++ The client name
++ The OAuth2 client id
+ 
+An example:
 ```
-[
-    {
-        clientName: "Client A",
-        oauth2ClientId: "client_A"
+{
+    "onboardedSolutionClient": [{
+        "clientName": "Windows Magnifier",
+        "solutionId": "net.gpii.windows.magnifier"
     },
     ...
-]
+    ],
+    "webPrefsConsumerClient": [{
+        "clientName": "Service A",
+        "oauth2ClientId": "org.chrome.cloud4chrome"
+    },
+    ...
+    ]
+}
 ```
 
-#### grantAuthorizationCode(userId, clientId, redirectUri, selectedPreferences)
-* **description**: Grant an Authorization Code for the specified user, client, redirect URI and selected preferences. We first check to see if we have an existing authorization decision for the user, client, and redirect URI. If we do, we issue a new code for that decision. Otherwise we create a new authorization decision record and a new code.
+#### grantGpiiAppInstallationAuthorization(gpiiToken, clientId)
+* **description**: Grant an authorization to a GPII app installation. The authorization allows a GPII app installation to access user settings associated with the given GPII token. The scenarios are handled by this function: 
+    * If the given GPII app installation has never been assigned an authorization for the given GPII token, the function will generate and return one.
+    * If the given GPII app installation has already been assigned an authorization for the given GPII token, and this token has not expired, the function will return this existing authorization;
+    * If the given GPII app installation has already been assigned an authorization for the given GPII token, but this token has already expired, the fuction will generate and return a new one.
 * **parameters:** 
-    * userId: Number. A system generated unique number that identifies the user.
-    * clientId: Number. A system generated unique number that identifies the client.
-    * redirectUri: String. The client redirection URI that the authorization server directs the user-agent to when a authorization decision is established.
-    * selectedPreferences: Object. An object specifying the preferences that the user has selected to share, in the privacy ontology. An example:
+    * gpiiToken: String. A GPII token that associates with user preferences.
+    * clientId: String. A system generated unique string that identifies the client.
+* **return:** Object. Contains an access token and the number of seconds that the access token will expire. For example:
+```
+{
+    "accessToken": "8ea3457bf283db5d34ea5a4079fa36b2",
+    "expiresIn": 3600
+}
+```
+Return an object that contains the error message and http status code if,
+    * The GPII token is wrong, or,
+    * The client is not a GPII App Installation Client.
```

Comment:
These 2 items are not being rendered as bullets. Do they need to be moved over to the start of the line?